lbry/lbry/stream/managed_stream.py

```diff
@@ -381,7 +381,7 @@ def _write_decrypted_blob(handle: typing.IO, data: bytes):
             os.mkdir(self.download_directory)
         self._file_name = await get_next_available_file_name(
             self.loop, self.download_directory,
-            file_name or self.descriptor.suggested_file_name
+            file_name or self.file_name
```

Comment:
`file_name or sanitize_file_name(self.file_name)`

Reply:
In `download_stream_from_uri()`, `ManagedStream` is first instantiated with input `file_name` and then `save_file()` is called without `file_name` argument. If the the user calls tries to call `jsonrpc_get` with some file name, they will get the sanitized suggested file name. Changing this on the callers side makes the code less intuitive and readble, so I will make this line `file_name or self._file_name or sanitize_file_name(self.descriptor.suggested_file_name)`.
https://github.com/lbryio/lbry-sdk/blob/master/lbry/lbry/stream/stream_manager.py#L416